### DISCUSSION TITLE
Tasty Target skill still working as passive long-range sonar

### DISCUSSION THREAD

**Discussion by Xardion495** [FakeFishGames/Barotrauma#14386] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I tried to attract attention without [duplicating tickets](https://github.com/FakeFishGames/Barotrauma/issues/13676)

The mentioned skill still working as legal cheat and killing all suspense. Please fix it

https://github.com/user-attachments/assets/e3e05692-7652-4dda-ac47-422cf43790e9

### Reproduction steps

1. Get 'Tasty Target' skill
2. Look at the terminal
3. Watch the rocks and enemies

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
It seems the fix we implemented previously got lost due to a mishap when merging branches, should be fixed (again) now. Commit in our private repo: https://github.com/FakeFishGames/Barotrauma-development/commit/495ef2494952e2cdbac84b1779ea770a4a599c64.

The fix should be coming up in the next update, sorry for the inconvenience!
